rtcp.c

```diff
@@ -143,6 +143,12 @@ guint32 janus_rtcp_get_receiver_ssrc(char *packet, int len) {
 				janus_rtcp_fb *rtcpfb = (janus_rtcp_fb *)rtcp;
 				return ntohl(rtcpfb->media);
 			}
+			case RTCP_PSFB: {
+				/* PSFB, Payload-specific FB message (rfc4585) */
+				janus_rtcp_fb *rtcpfb = (janus_rtcp_fb *)rtcp;
+				/* Fallback to media SSRC if sender SSRC is not set */
+				return ntohl(rtcpfb->ssrc) || ntohl(rtcpfb->media);
```

Comment:
I don't think this is correct: `rtcpfb->ssrc` is the SSRC of the packet sender (https://datatracker.ietf.org/doc/html/rfc4585#section-6.1), so IMO it's wrong to return that in a method that looks for the receiver SSRC. We do return `rtcpfb->ssrc` for RTCP_PSFB already, but in `janus_rtcp_get_sender_ssrc`. As such, I'd only leave `rtcpfb->media` in there, as we do for RTCP_RTPFB just a few lines above which uses the same struct.

If that works, it probably does make sense for 1.x as well since they share most of this RTCP code.